### PR TITLE
API 5.0 WP 3 (Working with Groups)

### PR DESCRIPTION
**Pull request by Bibo-Joshi**

### CODE REVIEW

File: telegram/bot.py [hunk 1]
```diff
@@ -2098,6 +2103,9 @@ def unban_chat_member(
         """
         data: JSONDict = {'chat_id': chat_id, 'user_id': user_id}
 
+        if only_if_banned is not None:
```
Comment (Poolitzer) [inline telegram/bot.py:2106]: now `is not None` is fine? :D

**Review by Poolitzer — COMMENTED** [review]
LGTM

File: telegram/bot.py (hunk 1 above)
Comment (Bibo-Joshi) [reply telegram/bot.py:2106]: see WP 1 :)

File: telegram/chat.py [hunk 2]
```diff
@@ -56,6 +59,11 @@ class Chat(TelegramObject):
         sticker_set_name (:obj:`str`): Optional. For supergroups, name of Group sticker set.
         can_set_sticker_set (:obj:`bool`): Optional. :obj:`True`, if the bot can change group the
             sticker set.
+        linked_chat_id (:obj:`int`): Optional. Unique identifier for the linked chat, i.e. the
+            discussion group identifier for a channel and vice versa; for supergroups and channel
+            chats. Returned only in :meth:`telegram.Bot.get_chat`.
+        location (:class:`telegram.ChatLocation`): Optional. For supergroups, the location to which
```
Comment (harshil21) [inline telegram/chat.py:65]: That 'Optional' should be inside the brackets right? Same for bio.

Comment (Bibo-Joshi) [reply telegram/chat.py:65]: Only for the arguments, not for the attributes. so in the change below, but thanks for the catch :)

**Pull request merged**